tldr/cross_file_calls/scanner.py

```
@@ -0,0 +1,306 @@
+
+
+def _build_call_relationships(call_graph: ProjectCallGraph) -> None:
+    """
+    Build relationships between calls and definitions.
+    
+    TODO: Full implementation is in resolver.py.
+    This is a placeholder that marks the graph as processed.
+    """
+    # The actual cross-file resolution is done in builder._resolve_cross_file_calls
+    # This function is kept for API compatibility
+    pass
```

Ruff (0.14.11)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)